p2panda-discovery/Cargo.toml

```diff
@@ -2,6 +2,16 @@
 name = "p2panda-discovery"
 version = "0.1.0"
 edition = "2021"
+authors = [
+  "adz <x12@adz.garden>",
+  "sandreae <contact@samandreae.com>",
+  "glyph <glyph@mycelial.technology>"
+]
+description = "Peer discovery traits and services"
+repository = "https://github.com/p2panda/p2panda"
+license = "MIT OR Apache-2.0"
+readme = "README.md"
+categories = ["discovery", "mdns"]
```

Comment:
Are these tags predefined by crates.io or free-form?

Reply:
These are keywords which I have selected, and which have already been used by other projects on crates.io.

I used `keyword: discovery` as a search term on crates.io to confirm.

Reply:
Oh wait...those are keywords and not categories.

There's a list of 55 categories here: https://crates.io/categories

I'll make change the `categories` field to `keywords` and add proper categories if any are a good fit.

Reply:
In any case, I don't think including categories is very important.